keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -697,43 +705,58 @@ generatedFrom:
   apiGroup: dra.example.com
   uid: foobar-uid
 
-vendorParameters:
-  # A vendor can put any kind of object here to pass the configuration
-  # parameters down to the kubelet plugin. In this case, the vendor
-  # driver simply copied the entire CR. It could also be some
-  # separate, smaller configuration type.
+# Setup parameters can be provided at two different levels:
+# - for all resources (here)
+# - for a single resource (not shown below)
+#
+# At the moment, only setup parameters defined by a vendor
+# are supported. In-tree definition of common setup parameters
+# might get added in the future.
+setup:
+  vendor:
```

Comment:
What is this `vendor` field? It is at the same level as the fields below, but is not set to anything...